packages/mui-material-next/src/LinearProgress/linearProgressClasses.ts

```diff
@@ -52,20 +64,26 @@ const linearProgressClasses: LinearProgressClasses = generateUtilityClasses('Mui
   'root',
   'colorPrimary',
   'colorSecondary',
+  'colorTertiary',
   'determinate',
   'indeterminate',
   'buffer',
   'query',
+  'fourColor',
   'dashed',
   'dashedColorPrimary',
   'dashedColorSecondary',
+  'dashedColorTertiary',
```

Comment:
For v6, we will remove composed classes: https://github.com/mui/material-ui/blob/master/packages/mui-material-next/migration.md#remove-composed-css-classes-and-styleoverrides-keys

So you don't need to add this one. I'm unsure about the `bar...` ones, what do you think? If those are achievable by composing classes, we shouldn't add them.

Reply:
@DiegoAndai Yes, I think it can also be done by composing classes, so I will fix this.

Reply:
I think best for the bars is to have the classes:
- `bar`
- `bar1`
- `bar2`

Reply:
I would use bar1 and bar2 for consistency with the existing classes. You don't need to make that change in this PR, though. If you're going to, please don't forget to add the removed classes to the migration guide.